srtcore/common.h

```diff
@@ -685,7 +685,7 @@ class StatsLossRecords
 
     void add(int32_t lo, int32_t hi)
     {
-        int32_t end = lo + CSeqNo::seqcmp(hi, lo);
+        int32_t end = CSeqNo::incseq(hi);
```

Comment:
Was `lo + (hi - lo)` for the general case (`(abs(seq1 - seq2) < m_iSeqNoTH`).
Now: `hi + 1`

Function `void add(int32_t lo, int32_t hi)` is never used? As well as class `StatsLossRecords`.
Changes in `common.h` do not effect the work of SRT.

Reply:
This is still an existing code. Not sure what to do with it, but even if it had to be used for testing purposes only, it should work correctly. I don't even know anymore what the API requirements for the `add` function were, I simply guessed it requires to get the range of sequences, with both ends included. Therefore the `end` sequence should be the first value for which the loop condition shouldn't be executed and it should stop there, so I defined it simply according to this statement.